FWCore/Framework/interface/OccurrenceTraits.h

```diff
@@ -424,6 +425,7 @@ namespace edm {
     using MyPrincipal = ProcessBlockPrincipal;
     using Context = GlobalContext;
     static bool constexpr isEvent_ = false;
+    static Transition constexpr transition_ = Transition::BeginProcessBlock;
```

Comment:
Is this BeginProcessBlock or is it InputProcessBlock?

Reply:
Fixed now with the new enum value.